secure.go

```diff
@@ -49,10 +50,12 @@ type Options struct {
 	CustomFrameOptionsValue string
 	// If ContentTypeNosniff is true, adds the X-Content-Type-Options header with the value `nosniff`. Default is false.
 	ContentTypeNosniff bool
-	// If BrowserXssFilter is true, adds the X-XSS-Protection header with the value `1; mode=block`. Default is false.
-	BrowserXssFilter bool
+	// If BrowserXSSFilter is true, adds the X-XSS-Protection header with the value `1; mode=block`. Default is false.
+	BrowserXSSFilter bool
```

Comment:
This has bugged me for a long time, and you are correct in that it should be `BrowserXSSFilter`. But we can not change it. This change would require a new version of the API as it breaks compatibility. Please revert!